daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -194,7 +198,7 @@ class DFDLTestSuite private[tdml] (
     defaultImplementationsDefault: Seq[String] = Runner.defaultImplementationsDefaultDefault,
     shouldDoErrorComparisonOnCrossTests: Boolean = Runner.defaultShouldDoErrorComparisonOnCrossTests,
     shouldDoWarningComparisonOnCrossTests: Boolean = Runner.defaultShouldDoWarningComparisonOnCrossTests) =
-    this(null, aNodeFileOrURL, validateTDMLFile, validateDFDLSchemas, compileAllTopLevel,
+    this(null, aNodeFileOrURL, useTDMLImplementation, validateTDMLFile, validateDFDLSchemas, compileAllTopLevel,
```

Comment:
You're right, this public deprecated constructor isn't called by Daffodil anymore so I will remove it.  To avoid breaking backward compatibility in case there's any external code which still calls this deprecated constructor, I will remove the extra `val __nl: Null,` parameter from the other constructor.  We can make the other constructor public if we need to, but I think Java code can call a Scala private constructor.